snuba/cli/subscriptions.py

```diff
@@ -119,7 +119,7 @@ def subscriptions(
     dataset = get_dataset(dataset_name)
 
     if not bootstrap_servers:
-        storage = dataset.get_writable_storage()
+        storage = dataset.get_entity().get_writable_storage()
```

Comment:
Do you plan to adapt subscriptions to the dataset model that includes multiple entities or are we missing this part from the plan? Now subscriptions reason in datasets assuming no entities. Now we need to make them deal with an entity and not a dataset as main entry point.

Reply:
I think subscriptions will still use the dataset as an entry point, since I assume they will also need to support JOIN queries. If not, we can move them to use the entity directly.

If subscriptions can be done on a JOIN query, then we will need to add that to the plan.

Reply:
So I think we need a plan for them i neither case. Either we make the subscription use an entity or, if they need joins, the subscriptions consumer needs a redesign because it needs to know which entity it has to follow (assuming it will follow one).
Is this part of your task ?

Reply:
No it's not. I'll add a subtask to JOINs to track this, really we need to follow up with Workflow to see if this is a feature they need/want and go from there.